src/odemis/gui/cont/tabs.py

```diff
@@ -499,6 +501,29 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 self
             )
 
+    def _on_overview_visible(self, val):
+        """
+        Apply the visibility status of overview streams in the Localisation tab to
+        the overview streams in the Chamber tab.
+        """
+        # All the overview streams
+        acquired_streams = set()
+        acquired_streams.update(self.tab_data_model.overviewStreams.value)
+        # Visible overview streams
+        streams = self.tab_data_model.views.value[0].stream_tree.streams
+        list_streams = []
+        for st in streams:
+            list_streams.append(st.stream)
+        # Invisible overview streams
+        unused_streams = acquired_streams.difference(set(list_streams))
+
+        chamber_tab = self.main_data.getTabByName("cryosecom_chamber")
+        # Hide the invisible overview streams
```

Comment:
This removes / adds the stream to the chamber tab, not just toggles the visible. It might be better to add another function to just toggle the visibility in the chamber overview?

Reply:
added a new function which is used at this place and also in line 2911 at reset_project_data